tools/wakebox/main.cpp

```diff
@@ -193,7 +193,7 @@ int run_batch(const char *params_path, bool has_output, bool use_stdin_file, boo
   }
 
   // Open the output file
-  int out_fd = open(result_path, O_WRONLY | O_CREAT | O_CLOEXEC, 0664);
+  int out_fd = open(result_path, O_WRONLY | O_CREAT | O_CLOEXEC | O_TRUNC, 0664);
```

Comment:
Small nit for future PRs, I think its better to create a new inode than it is to truncate. The only way I know to do this is to create a temp file and rename it into place.